src/controllers/controllerinputmappingtablemodel.cpp

```diff
@@ -197,6 +197,11 @@ QVariant ControllerInputMappingTableModel::data(const QModelIndex& index,
         }
 
         const MidiInputMapping& mapping = m_midiInputMappings.at(row);
+
+        if (holds_alternative<QJSValue>(mapping.control)) {
+            return "";
+        }
+
```

Comment:
I don't really understand what that function does so I just opted to not execute it if the control is a lambda.